### PR TITLE
Scripting: fix generateContextDoc path and url

### PR DESCRIPTION
**Pull request by stu-elastic**
* Add doc runtime class path
* Use `getAllHttpSocketURI.get(0)` instead of `getAllHttpSocketURI` to get a single
  test cluster URL rather than a list

### CODE REVIEW

File: docs/painless/painless-api-reference/painless-api-reference-string-sort/packages.asciidoc [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+* static double nullSafeSortNumeric(Number)
+* static String nullSafeSortString(Object)
+* static Boolean nulleq(Object, Object)
+* static Boolean or(Boolean, Boolean)
+* static Boolean regex(String, String)
+* static Boolean startsWith(String, String, Boolean)
+* static Number sub(Number, Number)
+* boolean {java11-javadoc}/java.base/java/lang/Object.html#equals(java.lang.Object)[equals](Object)
+* int {java11-javadoc}/java.base/java/lang/Object.html#hashCode()[hashCode]()
+* String {java11-javadoc}/java.base/java/lang/Object.html#toString()[toString]()
```
Comment (jdconrad) [inline docs/painless/painless-api-reference/painless-api-reference-string-sort/packages.asciidoc:58]: Please add this to the exclusions list. Check out `isInternalClass` as part of ContextDocGenerator. We try to exclude other internal methods here. There isn't currently a better way to do this.

File: docs/painless/painless-api-reference/painless-api-reference-string-sort/packages.asciidoc [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+* static Boolean endsWith(String, String, Boolean)
+* static Integer indexOf(String, String, Number, Boolean)
+* static Integer length(String)
+* static Number number(String, Number)
+* static String string(Object)
+* static Boolean stringContains(String, String, Boolean)
+* static String substring(String, Number, Number)
+* boolean {java11-javadoc}/java.base/java/lang/Object.html#equals(java.lang.Object)[equals](Object)
+* int {java11-javadoc}/java.base/java/lang/Object.html#hashCode()[hashCode]()
+* String {java11-javadoc}/java.base/java/lang/Object.html#toString()[toString]()
```
Comment (jdconrad) [inline docs/painless/painless-api-reference/painless-api-reference-string-sort/packages.asciidoc:22]: Please exclude these as well. These really are only meant to be used internally.

File: docs/painless/painless-api-reference/painless-api-reference-shared/packages.asciidoc [hunk 3]
```diff
@@ -8642,6 +8655,14 @@ See the <<painless-api-reference-shared, Shared API>> for a high-level overview
...
 
 
+[[painless-api-reference-shared-ScoreScript-ExplanationHolder]]
+==== ScoreScript.ExplanationHolder
+* boolean {java11-javadoc}/java.base/java/lang/Object.html#equals(java.lang.Object)[equals](Object)
+* int {java11-javadoc}/java.base/java/lang/Object.html#hashCode()[hashCode]()
+* void set(String)
+* String {java11-javadoc}/java.base/java/lang/Object.html#toString()[toString]()
+
+
```
Comment (jdconrad) [inline docs/painless/painless-api-reference/painless-api-reference-shared/packages.asciidoc:8665]: More to exclude.

File: docs/painless/painless-api-reference/painless-api-reference-number-sort/packages.asciidoc [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+* static String nullSafeSortString(Object)
+* static Boolean nulleq(Object, Object)
+* static Boolean or(Boolean, Boolean)
+* static Boolean regex(String, String)
+* static Boolean startsWith(String, String, Boolean)
+* static Number sub(Number, Number)
+* boolean {java11-javadoc}/java.base/java/lang/Object.html#equals(java.lang.Object)[equals](Object)
+* int {java11-javadoc}/java.base/java/lang/Object.html#hashCode()[hashCode]()
+* String {java11-javadoc}/java.base/java/lang/Object.html#toString()[toString]()
+
```
Comment (jdconrad) [inline docs/painless/painless-api-reference/painless-api-reference-number-sort/packages.asciidoc:59]: More to exclude.

**Review by jdconrad — CHANGES_REQUESTED** [review]
Please take a look at `ContextDocGenerator.isInternalClass` and add exclusions for the new internal classes for query langs like EQL.

**Review by jdconrad — APPROVED** [review]
LGTM. Thanks!

File: docs/painless/painless-api-reference/painless-api-reference-string-sort/packages.asciidoc (hunk 1 above)
Comment (stu-elastic) [reply docs/painless/painless-api-reference/painless-api-reference-string-sort/packages.asciidoc]: Excluded.

File: docs/painless/painless-api-reference/painless-api-reference-string-sort/packages.asciidoc (hunk 2 above)
Comment (stu-elastic) [reply docs/painless/painless-api-reference/painless-api-reference-string-sort/packages.asciidoc]: Excluded.

File: docs/painless/painless-api-reference/painless-api-reference-shared/packages.asciidoc (hunk 3 above)
Comment (stu-elastic) [reply docs/painless/painless-api-reference/painless-api-reference-shared/packages.asciidoc]: Excluded.

File: docs/painless/painless-api-reference/painless-api-reference-number-sort/packages.asciidoc (hunk 4 above)
Comment (stu-elastic) [reply docs/painless/painless-api-reference/painless-api-reference-number-sort/packages.asciidoc]: Excluded.

**Comment by stu-elastic** [comment]
master: 3057e0f
7.x (7.10): c491212

**Pull request merged**